### ISSUE TITLE
Accessibility: allow greater zoom levels

### ISSUE THREAD

**Issue by hackergrrl** [deltachat/deltachat-desktop#5558]
- Operating System (Linux/Mac/Windows/iOS/Android):
- Delta Chat Version: 1.46.1 (git: v1.33.0-881-g899f196d)
- Expected behavior: Able to zoom past 1.4x
- Actual behavior: Unable to do so
- Steps to reproduce the problem: Try to zoom past 1.4x

I've been dealing with vision impairment, and the 1.4x max zoom on the desktop client just isn't enough for me to be able to read messages. Ctrl-+ and Ctrl-- don't affect zoom level. It would be great if they did, and that there be no imposed maximum zoom.

Thanks for DeltaChat!

**Comment by ghost** [comment]
I upgraded to 2.15.0 and this issue is delightfully resolved. Thanks!
